src/oracle_test/regress/sql/ora_plisql.sql

```diff
@@ -960,3 +960,157 @@ select * from ftest(null::t1);
 
 drop function ftest;
 drop table t1;
+
+
+create or replace procedure protest 
+as 
+begin 
+raise notice 'protest';
+end;
+/ 
+
+CALL protest;  -- should failed 
+CALL protest();
+exec protest;      
+exec protest();
+drop  procedure protest ;
+
+create or replace procedure procedure_addtest (a int, b int)
+as 
+begin 
+raise notice 'a + b = %', a + b;
+end;
+/ 
+
+exec procedure_addtest(12,13);
+drop procedure procedure_addtest;
+
+--
+-- Tests for procedures / exec syntax
+--
+CREATE or replace PROCEDURE test_proc1
+AS 
+BEGIN
+    NULL;
+END;
+/
+
+CALL test_proc1;  -- should failed 
```

Comment:
EXEC:
1、exec procedure; 
2、exec procedure(); 
3、exec procedure(12,13); 

CALL:
1、call procedure(); 
2、call procedure(12,13); 

Using CALL to call a stored procedure without arguments must be parenthesized, otherwise the call should fail, but EXEC is different here.